manifests/params.pp

```diff
@@ -261,6 +282,7 @@
     'Debian':  { $package_provider = 'dpkg' }
     'Solaris': { $package_provider = 'sun' }
     'Suse':    { $package_provider = 'rpm' }
+    'FreeBSD': { $package_provider = 'pkgng' }
```

Comment:
Is setting this required? in 99% of the cases undef is fine and puppet will figure it out automatically.

Reply:
I just added it for completeness, but it looks like there is code depending on it in https://github.com/voxpupuli/puppet-splunk/blob/99442a75bce646ae8795a620f2b0c2635eceec38/manifests/forwarder/install.pp#L12 
To be honest, I did not test it without setting is specifically.